Sources/ComposableArchitecture/SwiftUI/WithViewStore.swift

```diff
@@ -186,12 +186,71 @@ public struct WithViewStore<ViewState, ViewAction, Content> {
 // MARK: - View
 
 extension WithViewStore: View where Content: View {
-  /// Initializes a structure that transforms a store into an observable view store in order to
-  /// compute views from state.
+  /// Initializes a structure that transforms a ``Store`` into an observable ``ViewStore`` in order
```

Comment:
I wanted a short explanation of the performance pitfalls directly in the docs for `WithViewStore.init(_:observe:)` so that it's as close to the API as possible. Bummer that it has to be repeated a few times, but I think worth it.